### DISCUSSION TITLE
It's time to move Darwinia into the RingDAO

### DISCUSSION THREAD

**Discussion by boundless-forest** [ringecosystem/discussion#9] [General]
In the last several months, we have put a lot of effort into integrating projects in our ecosystem into the RingDAO scope. Most of the work has been completed now, including migrating domains, GitHub repos, websites, and documentation, except for the Darwinia staff. This is because Darwinia has been running for a long time and is a well-known brand in the broader communities, so we delayed the migration. 

But now I believe it's time to propose a swift migration to the RingDAO, encompassing the GitHub repositories, domains, and sites. This can help all members lower the barrier for new users to understand RingDAO or Darwinia. It will unify our community around a common development goal and accelerate our progress. This task can be addressed either now or later, so I think we shouldn't delay too much. I look forward to embarking on the journey with a vibrant and active RingDAO community.

**Comment by hackfisher** [comment]
After discussions, it was clarified that these changes primarily involve domain names and GitHub. The education team has concluded that these changes do not impact the brand or project structure and can be addressed through certain integrations. At this point, I am inclined to retain these domain names and Darwinia sub-brands and domains.
